draft-ietf-jose-hpke-encrypt.md

```diff
@@ -149,8 +148,7 @@ The HPKE KEM, KDF, and AEAD used depend on the JOSE-HPKE algorithm used.
 
 HPKE supports several modes, which are described in Table 1 of {{RFC9180}}.
 
-In JOSE-HPKE, the HPKE mode used (e.g, "mode_base" or "mode_auth_psk") is determined
-by the presence of the JOSE Header parameters "psk_id" and "auth_kid".
+In JOSE-HPKE, the "base_mode" and "mode_psk" are determined by the presence of the JOSE Header parameter "psk_id"
```

Comment:
```suggestion
In JOSE-HPKE, only "mode_base" and "mode_psk" are supported.
When "psk_id" JOSE Header parameter is present the mode is "mode_psk", otherwise the mode is "mode_base".
```